### PR TITLE
[WIP]Object store container WT and Collection

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
As for 5.8 Storage speration..
- rename  cfme/storage/object_store.py >  cfme/storage/object_store_container.py
- rename   cfme/tests/storage/test_object_store.py >  cfme/tests/storage/test_object_store_container.py

{{pytest: cfme/storage/object_store_container.py -v --use-provider rhos7-ga}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3dbda16eefaacfa878774e3efdbc257880d5c6d9:

`cfme/storage/object_store_container.py`:
- :bangbang: Line 17:37: **E241** *multiple spaces after ','*
- :bangbang: Line 107:38: **E231** *missing whitespace after ','*
- :bangbang: Line 117:36: **E241** *multiple spaces after ','*
- :bangbang: Line 136:1: **E302** *expected 2 blank lines, found 1*

`cfme/tests/storage/test_object_store_container.py`:
- :bangbang: Line 15:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 19:5: **E265** *block comment should start with '# '*
- :bangbang: Line 22:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 26:5: **E265** *block comment should start with '# '*
- :bangbang: Line 31:5: **E265** *block comment should start with '# '*
- :heavy_exclamation_mark: Line 34:87: **W292** *no newline at end of file*

Hmmm
:frowning: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -2651,12 +2651,15 @@ class ListView(EntitiesConditionalView):
...
         @property
         def entity_names(self):
-            """ looks for entities and extracts their names
+            """ looks for entities and extracts their names or keys
 
             Returns: all current page entities
             """
             try:
-                return [row.name.text for row in self.elements.rows()]
+                try:
```
Comment (vatsalparekh) [inline widgetastic_manageiq/__init__.py]: I don't think a `try` inside `try` in required, you can simply write multiple `except` and catch accordingly

Comment (vatsalparekh) [inline widgetastic_manageiq/__init__.py]: I don't think a try inside try in required, you can simply write multiple except and catch accordingly

**Comment by dajoRH** [comment]
Lint report for commit f75489a3bba5a5b16a587d6a2d03a7b0922db327:

`cfme/tests/storage/test_object_store_container.py`:
- :bangbang: Line 23:1: **E302** *expected 2 blank lines, found 1*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request closed without merging**